app/services/file_workflow_service.py

```
@@ -334,28 +342,49 @@ def _scan_path(self, path: MonitoredPath, db: Session) -> dict:
-                    "to_cold": [],
-                    "to_hot": [],
-                    "inventory_updated": 0,
-                    "skipped_hot": 0,
-                    "skipped_cold": 0,
-                }
+            local_symlink_locations = []
+            for location in path.storage_locations:
+                try:
+                    backend = get_backend(location)
+                except Exception:
+                    continue
```

Ruff (0.15.10)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)